en/04_Changelogs/5.0.0.md

```diff
@@ -573,8 +573,8 @@ This is a major release and contains many breaking API changes. Deprecation warn
 - Removed deprecated method `SilverStripe\Admin\ModelAdmin::SearchForm()`
 - Removed deprecated method `SilverStripe\Admin\ModelAdmin::SearchSummary()`
 - Removed deprecated method `SilverStripe\Admin\SecurityAdmin::Backlink()`
-- Removed deprecated method `SilverStripe\Admin\SecurityAdmin::Breadcrumbs()`
-- Removed deprecated method `SilverStripe\Admin\SecurityAdmin::getEditForm()`
+- Removed deprecated method `SilverStripe\Admin\SecurityAdmin::Breadcrumbs()`. Class `SilverStripe\Admin\SecurityAdmin` inherits this method from `SilverStripe\Admin\ModelAdmin`
+- Removed deprecated method `SilverStripe\Admin\SecurityAdmin::getEditForm()`. Class `SilverStripe\Admin\SecurityAdmin` inherits this method from `SilverStripe\Admin\ModelAdmin`
```

Comment:
```suggestion
```
Just don't mention them. The method signatures are the same, and you can still call or override them the same way. Nothing broke, so we don't need to mention it.

Reply:
Removed these lines 👍